### ISSUE TITLE
Bug: A 'NullReferenceException' Is Thrown for NULL Query Expression

### ISSUE THREAD

**Issue by mikependon** [mikependon/RepoDB#600]
As reported by the user, a NullReferenceException is thrown if the Query Expression is equate to a NULL values.

```csharp
connection.Query<Person>(e => e.Name == null);
```

It has happened on the latest compiler, in which such basic scenario is apparently not covered by the Integration Tests. We have added the below's Integration Test.

```csharp
[TestMethod]
public void TestSqlConnectionQueryAsyncViaExpressionWithNullValue()
{
    // Setup
    var tables = Helper.CreateIdentityTables(10);
    var last = tables.Last();

    using (var connection = new SqlConnection(Database.ConnectionStringForRepoDb))
    {
        // Act
        connection.InsertAll(tables);

        // Act
        var result = connection.QueryAsync<IdentityTable>(c => c.ColumnNVarChar == null).Result;

        // Assert
        Assert.AreEqual(0, result.Count());
    }
}
```

**Comment by mikependon** [comment]
@eriego - the fixes has been pushed to [RepoDb v1.12.2](https://www.nuget.org/packages/RepoDb/1.12.2).

**Comment by eriego** [comment]
It works.

Thank you.
